test/e2e/apimachinery/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - sig-apimachinery-maintainers
+approvers:
+  - sig-apimachinery-maintainers
```

Comment:
this line should be:
```
  - sig-api-machinery-api-approvers
```

Reply:
Updated in https://github.com/kubernetes/kubernetes/pull/86854/commits/1fa8ddd8aa6e347fe6a6500f74dba6310dd68293